source/fundamentals/connection.txt

```diff
@@ -143,11 +143,32 @@ URI to specify the behavior of the client.
      - Specifies the logger level used by the driver. Valid choices are:
        ``error``, ``warn``, ``info``, and ``debug``.
 
+   * - **maxPoolSize**
+     - integer
+     - ``10``
+     - **This option requires that you set the useUnifiedTopology option to true**.
```

Comment:
```suggestion
     - You must set the ``useUnifiedTopology`` option to **true** to use this option.
```

Reply:
This also applies to the other instances.

Reply:
Thanks for the note. I agree that this phrasing is more linear, more imperative, and easier to understand. Will try and keep this in mind going forward.

Fix: https://github.com/mongodb/docs-node/pull/149/commits/e1cd138fc81b04e757e426e99a2027e8d6a69905